nautobot/core/management/commands/generate_test_data.py

```diff
@@ -292,7 +292,7 @@ def handle(self, *args, **options):
             self.stdout.write(
                 self.style.WARNING(f'Flushing all existing data from the database "{options["database"]}"...')
             )
-            call_command("flush", "--no-input", "--database", options["database"])
+            call_command("flush", "--no-input", "--database", options["database"], inhibit_post_migrate=True)
```

Comment:
Not familiar with `inhibit_post_migrate` - what's it do, why's it needed? :-)

Reply:
Before merging main PR into next, I'll separate fixes not directly related to controller models into distinct PRs :-)

Reply:
Removed this fix, as it breaks the current tests.